util/prefix_validate.go

```diff
@@ -0,0 +1,35 @@
+package util
```

Comment:
I understand why you moved the file to the `util` package, and I'm basically fine with this.

However, I think we should adjust a few things:

1. The function is still called `IsValidTagPrefix`, although it is not related directly to Git tags any more. Shouldn't it be `IsValidVersionPrefix` then?
2. If we change the name like this, I'd suggest to name the file `version_prefix.go`, since it then contains all the helper methods for version prefixes.